h/api/storage.py

```diff
@@ -61,12 +74,77 @@ def create_annotation(request, data):
     :returns: the created annotation
     :rtype: dict
     """
-    annotation = elastic.Annotation(data)
-
-    # FIXME: this should happen when indexing, not storing.
-    _prepare(request, annotation)
+    if not request.feature('postgres_write'):
+        return _legacy_create_annotation_in_elasticsearch(request, data)
```

Comment:
I think this is similar to a comment on another one of these 4 pull requests. This would mean that when the `postgres_write` feature flag is on, we only write to Postgres. But we decided to write to both Postgres and the old Elasticsearch. Or is this being handled somewhere else that I don't understand yet?